composites/Plugin/Shared/components/Collapsible.js

```diff
@@ -152,23 +144,33 @@ CollapsibleStateless.propTypes = {
 	Heading: PropTypes.func,
 	isOpen: PropTypes.bool.isRequired,
 	onToggle: PropTypes.func.isRequired,
-	prefixIcon: PropTypes.string,
-	prefixIconCollapsed: PropTypes.string,
-	prefixIconColor: PropTypes.string,
-	prefixIconSize: PropTypes.string,
+	prefixIcon: PropTypes.shape( {
```

Comment:
Perhaps declare the shape seperately, and reuse this declaration wherever it's used?

Done in snippetEditorFields's descriptionLengthProgress, for instance. 

Not important enough that it has to be addressed now though.